vividus-tests/src/main/resources/data/tables/system/mobile_app/locators/ios.table

```diff
@@ -18,3 +18,5 @@
 |qrCodeContainerXpath       |//XCUIElementTypeOther[@name='RNE__Image__children__container'] |
 |firstItemAccessibilityId   |Item 1                                                          |
 |secondItemAccessibilityId  |Item 2                                                          |
+|secondVisibleItemXpath     |(//XCUIElementTypeOther[@name="Item 2 Text 2"])[1]              |
+|firstVisibleItemXpath      |(//XCUIElementTypeOther[@name="Item 1 Text 1"])[1]              |
```

Comment:
why is index used?

Reply:
<img width="1267" alt="ios" src="https://user-images.githubusercontent.com/18533593/198703037-5b055b05-faad-43a1-af42-379ead0697db.png">